pkg/prm/prm.go

```diff
@@ -0,0 +1,54 @@
+//nolint:structcheck,unused
+package prm
+
+import "github.com/Masterminds/semver"
+
+type Prm struct {
+	codeDir string
+	cache   []toolCache
+}
+
+type toolCache struct {
+	toolName string
+	tool     *Tool
+}
+type PuppetVersion struct {
+	version semver.Version
+}
+
+// Given a list of tool names, check if these are groups, and return
+// an expanded list containing all the toolNames
+func (*Prm) checkGroups(tools []string) []string {
+	// TODO
+	return []string{}
+}
+
+// Look within codeDir for a "validate.yml" containing
+// a list of tools and/or tool groups that should be run against
+// code within codeDir.
+func (*Prm) checkLocalConfig() []string {
```

Comment:
🤔 Should this be named something more semantic, like `checkLocalValidateList`?

Reply:
I would still look at validate.yml as a users local config, but yeah its probably best to describe the function.